core/kazoo_documents/src/kzd_vmboxes.erl.src

```diff
@@ -183,6 +184,18 @@ notify_email_addresses(Doc, Default) ->
 set_notify_email_addresses(Doc, NotifyEmailAddresses) ->
     kz_json:set_value([<<"notify_email_addresses">>], NotifyEmailAddresses, Doc).
 
+-spec oldest_message_first(doc()) -> boolean().
+oldest_message_first(Doc) ->
```

Comment:
These changes need to be migrated to the .erl version. This change is just to serve as a reminder.

Reply:
@jamesaimonetti this has been updated.  Please review :)